lib/bitcoin/lock/LockMonitor.ts

```diff
@@ -375,4 +408,12 @@ export default class LockMonitor {
 
     return lockInfoToSave;
   }
+
+  private async isUnlockTimeReached (unlockTransactionTime: number): Promise<boolean> {
+    const currentBlockTime = await this.bitcoinClient.getCurrentBlockHeight();
+
+    console.info(`Current block: ${currentBlockTime}; Current lock's unlock block: ${unlockTransactionTime}`);
+
+    return currentBlockTime - unlockTransactionTime >= 0;

```

Comment:
Nit:
```suggestion
    return currentBlockTime >= unlockTransactionTime;

```

Reply:
THANK you  ... I was completely blanking out for some reason and made it so much harder than it has to be :) I actually spent like 10/15 mins on this statement alone.